### PR TITLE
CLD-7906-titles-cut-off

### PR DESCRIPTION
**Pull request by tsi**

### CODE REVIEW

File: src/assets/styles/components/playlist.scss [hunk 1]
```diff
@@ -27,13 +27,22 @@
 
                 .cld-plw-item-info-wrap {
                     left: 150px;
-                    padding: 20px 15px;
+                    padding: 15px 10px;
                     top: 0;
+                    max-width: calc(100% - 150px);
```
Comment (mikeys) [inline src/assets/styles/components/playlist.scss]:
```
https://caniuse.com/#feat=calc
```

Let's make sure (with Amit) that we're ok with this.

Comment (tsi) [reply src/assets/styles/components/playlist.scss]: Any news about this? I think `calc()` is safe enough these days, and even if not supported nothing terrible should happen in this case...

Comment (gilnoycloudinary) [reply src/assets/styles/components/playlist.scss]: @tsi what about flex ?

Comment (tsi) [reply src/assets/styles/components/playlist.scss]: @gilnoycloudinary can do without flex/calc

File: src/assets/styles/components/playlist.scss [hunk 2]
```diff
@@ -27,13 +27,23 @@
 
                 .cld-plw-item-info-wrap {
                     left: 150px;
-                    padding: 20px 15px;
+                    padding: 15px 10px;
                     top: 0;
+                    right: 0;
+                    width: auto;
```
Comment (gilnoycloudinary) [inline src/assets/styles/components/playlist.scss]: @tsi no flex or calc is needed ?

Comment (tsi) [reply src/assets/styles/components/playlist.scss]: That's correct

**Comment by mikeys** [comment]
@tsi Approved from my end. @gilnoycloudinary ?

File: src/assets/styles/components/playlist.scss (hunk 2 above)
Comment (gilnoycloudinary) [reply src/assets/styles/components/playlist.scss]: @tsi great, merge!

**Pull request merged**